docker/server/Dockerfile

```
@@ -17,6 +17,8 @@ COPY docker/server/start_server.sh src/start_server.sh
 
 RUN /bin/bash -c "source /opt/ros/noetic/setup.bash; catkin config --cmake-args -DCMAKE_BUILD_TYPE=Release; catkin build"
 
+RUN rm -rf $(ccache -p | grep cache_dir | awk -F'=' '{print $2}' | xargs)
```

This could be simplified into:
```suggestion
RUN rm -r $(ccache -k cache_dir)
```
Also `--force` shouldn't be necessary in this case.

Alternatively
```suggestion
RUN ccache -C
```
would also work, but only cleans the cache and leaves 3.3mb behind.

---

Ah nice, I missed that, thanks.